### PR TITLE
Added unit tests

### PR DESCRIPTION
**Pull request by Hakky54**

### CODE REVIEW

File: kohttp-test/src/test/kotlin/io/github/rybalkinsd/kohttp/dsl/async/HttpDeleteAsyncDslTest.kt [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+                val parsedResponse = it.toJson()
+                assertContainsAtLeast(expectedHeader, parsedResponse["headers"])
+                assertContainsExactly(expectedParams, parsedResponse["args"])
+                assertContainsExactly(expectedForm, parsedResponse["form"])
+                assertThat(it.code()).isEqualTo(200)
+            }
+        }
+    }
+
+}
```
Comment (rybalkinsd) [inline kohttp-test/src/test/kotlin/io/github/rybalkinsd/kohttp/dsl/async/HttpDeleteAsyncDslTest.kt:70]: Hmm, we definitely need a linter
```suggestion
}

```

Comment (Hakky54) [reply kohttp-test/src/test/kotlin/io/github/rybalkinsd/kohttp/dsl/async/HttpDeleteAsyncDslTest.kt:70]: I also saw the same suggestions on different commits. What is the benefit of an additional line at the end of a class? Or is it your coding style?

**Comment by Hakky54** [comment]
By the way I initially thought that these unit tests aren't present so I added it. But then I discovered a different package which also has similar unit tests. Any idea why this is the case?

For example the HttpDeleteAsyncDslTest:
- kohttp/src/test/kotlin/io/github/rybalkinsd/kohttp/dsl/async/HttpDeleteAsyncDslTest.kt
- kohttp-test/src/test/kotlin/io/github/rybalkinsd/kohttp/dsl/async/HttpDeleteAsyncDslTest.kt

**Comment by rybalkinsd** [comment]
> Any idea why this is the case?

`:kohttp-test` depends on both `:kohttp` an `:kohttp-jackson` and any other module we will introduce in the future.

`.asJson` function from `:kohttp-jackson` simplifies tests.

**Comment by Hakky54** [comment]
Ah okay, that makes sense

**Pull request closed without merging**